wallet-new/src/Cardano/Wallet/API/Request/Filter.hs

```diff
@@ -195,6 +222,21 @@ instance ( HasServer subApi ctx
                           return $ toFilterOperations (parseQueryText $ rawQueryString req) (Proxy @params)
         in route (Proxy :: Proxy subApi) context delayed
 
+-- | Defines name of @FilterBy syms res@ as sum of @syms@,
+-- and specifies parameter 'FilterBy' provides.
+-- Used in e.g. logging.
+instance (KnownSymbols syms, syms ~ ParamNames res params) => ApiHasArgClass (FilterBy params res) where
+    type ApiArg (FilterBy params res) = FilterOperations res
+
+    apiArgName _ =
+        let filterNames = mconcat . intersperse ", " $ symbolVals (Proxy @syms)
+        in  formatToString ("filters: ("%build%")") filterNames
+
+-- | Defines how 'FilterBy' is logged by just refering to
+-- 'instance Buildable SortOperations'.
```

Comment:
Copy/paste wasn't fixed :)

Reply:
Not sure what you mean here. Not sure what this comment means either actually ^^"...

Reply:
I suppose @parsonsmatt says about `s/SortOperations/FilterOperations`.

I meant here that `toLogParamInfo` does just ~`sformat build` inside, it's really expressed askew though :thinking: